### ISSUE TITLE
Ceph kernel hang on shutdown

### ISSUE THREAD

**Issue by samcday** [samcday/home-cluster#30]
```
Sep 21 06:51:58 m710q-1 kernel: libceph: connect (1)172.31.52.146:6789 error -1
Sep 21 06:51:58 m710q-1 kernel: libceph: mon2 (1)172.31.52.146:6789 connect error
Sep 21 06:51:58 m710q-1 kernel: libceph: mon0 (1)172.31.198.228:6789 socket closed (con state V1_BANNER)
Sep 21 06:51:58 m710q-1 kernel: libceph: mon0 (1)172.31.198.228:6789 socket closed (con state V1_BANNER)
Sep 21 06:51:58 m710q-1 kernel: libceph: mon0 (1)172.31.198.228:6789 socket closed (con state V1_BANNER)
Sep 21 06:51:59 m710q-1 kernel: tasks_rcu_exit_srcu_stall: rcu_tasks grace period number 37 (since boot) gp_state: RTGS_POST_SCAN_TASKLIST is 11222989 jiffies old.
Sep 21 06:51:59 m710q-1 kernel: Please check any exiting tasks stuck between calls to exit_tasks_rcu_start() and exit_tasks_rcu_finish()
Sep 21 06:52:00 m710q-1 kernel: libceph: mon0 (1)172.31.198.228:6789 socket closed (con state V1_BANNER)
```

**Comment by samcday** [comment]
It's possible the issue here is Zincati was opting to yeet 2 of the machines at the same time, and Ceph was trying to hold off the reboot until it got mon quorum acks (which will never come because 2/3 nodes are now rebooting).

So this issue might end up being solved by #29 ...

**Comment by samcday** [comment]
I think the hang is caused by workloads that didn't properly unmount ceph rbd/fs volumes. It happens reliably if I randomly shut a machine down, but not if that machine is fully drained and kubelet stopped first.
